plone/app/contenttypes/browser/templates/document.pt

```diff
@@ -3,17 +3,20 @@
     xmlns:metal="http://xml.zope.org/namespaces/metal"
     xmlns:i18n="http://xml.zope.org/namespaces/i18n"
     lang="en"
-    metal:use-macro="context/main_template/macros/master"
+    metal:use-macro="context/@@main_template/macros/master"
     i18n:domain="plone">
 <body>
 
 <metal:content-core fill-slot="content-core">
 <metal:content-core define-macro="content-core"
-                    tal:define="toc context/table_of_contents|nothing;">
-  <div id="parent-fieldname-text"
-      tal:condition="context/text | nothing"
```

Comment:
@petschki @agitator This one had a reason to be. Any reason why do you remove it when you refactored it? Can we bring it back? Thanks.

Reply:
well, this was not removed, this was wrapped with <section id="section-text" /> @agitator you're the author of this ... any hints?

Reply:
I'm referring to the `condition = context/text | nothing`, the nothing was removed.

Reply:
no reason, was probably mishap ;-)